components/AppHeader.vue

```diff
@@ -201,9 +210,25 @@ export default class AppHeader extends Vue {
         wallet: accounts[0].address,
         method,
       })
-      return this.initWallet(connection)
+      await this.initWallet(connection)
+      if (!this.currentAddress || !this.signer) return
```

Comment:
`isLoading` is not set back to false in all `return` cases, consider using `finally`